distribution/cli.go

```diff
@@ -257,6 +257,12 @@ func releaseBundleDeleteCmd(c *cli.Context) error {
 
 	params := distributionServices.NewDeleteReleaseBundleParams(c.Args().Get(0), c.Args().Get(1))
 	params.DeleteFromDistribution = c.BoolT("delete-from-dist")
+	params.Sync = c.Bool("sync")
+	maxWaitMinutes, err := cliutils.GetIntFlagValue(c, "max-wait-minutes", 0)
```

Comment:
Shouldn't the last argument of the ```cliutils.GetIntFlagValue(c, "max-wait-minutes", 0)``` function should be 60?
See the default flafg value - 
```
--max-wait-minutes    [Default: 60] Max minutes to wait for sync distribution.
```